logs.txt

```diff
@@ -0,0 +1,4 @@
+Nov 26, 2020 9:11:40 PM models.LogToFileModel log
```

Comment:
Do not commit the logs.txt file

Reply:
it is from previous delivery but i delete now